scripts/sync.py

```diff
@@ -120,14 +122,16 @@ def main():
 
     # pause execution until
     if args.until:
-        logger.info('pausing execution until {0}'.format(args.until))
+        until = datetime.strptime('2017-01-01T15:00:00', '%Y-%m-%dT%H:%M:%S')
```

Comment:
I believe you want `datetime.strptime('args.until', '%Y-%m-%dT%H:%M:%S')`.

Reply:
6f7e4e7